grpc/src/main/java/com/linecorp/armeria/internal/server/grpc/AbstractServerCall.java

```diff
@@ -515,11 +515,11 @@ private void doSendHeaders(Metadata metadata) {
         responseHeaders = headers;
     }
 
-    final HttpData toPayload(O message) throws IOException {
+    public final HttpData toPayload(O message) throws IOException {
```

Comment:
```suggestion
    protected final HttpData toPayload(O message) throws IOException {
```

Reply:
Done in [fa4d628](https://github.com/line/armeria/commit/fa4d628b0a113437da1d9ebb625332dad3b299be)!